x-pack/plugins/security_solution/public/explore/users/components/kpi_users/authentications/index.tsx

```diff
@@ -79,7 +79,7 @@ const UsersKpiAuthenticationsComponent: React.FC<UsersKpiProps> = ({
     endDate: to,
     indexNames,
     startDate: from,
-    skip: querySkip,
+    skip: querySkip || isChartEmbeddablesEnabled,
```

Comment:
I happened to notice this was the only chart embeddables query without this condition, I think this is correct??

Reply:
Thanks for adding the check!